picasso/src/test/java/com/squareup/picasso/TestUtils.java

```diff
@@ -274,11 +274,11 @@ static Bitmap makeBitmap(int width, int height) {
     return Bitmap.createBitmap(width, height, ALPHA_8);
   }
 
-  static final OkHttp3Downloader UNUSED_DOWNLOADER = new OkHttp3Downloader(new Call.Factory() {
-    @Override public Call newCall(okhttp3.Request request) {
+  static final Call.Factory UNUSED_CALL_FACTORY = new Call.Factory() {
+    @Override public Call newCall(@NonNull okhttp3.Request request) {
```

Comment:
nit: remove @NonNull, let's document @Nullable with everything else defaulting to @NonNull.